src/helpers/handleFile.ts

```diff
@@ -0,0 +1,24 @@
+export default function (callback: (stringList: string) => void) {
+  const input = document.createElement('input')
+  input.type = 'file'
+  input.accept = '.txt,.csv'
+  input.multiple = false
+
+  input.onchange = ({ target }) => {
```

Comment:
This might look strange, but because we create an input, it always will be here, so destructuring wont fail